src/coord/src/catalog.rs

```diff
@@ -1426,10 +1437,10 @@ pub enum Op {
 #[derive(Debug, Clone)]
 pub enum OpStatus {
     CreatedDatabase(String, i64),
-    CreatedSchema,
+    CreatedSchema(String, String, i64),
```

Comment:
As a rule of thumb, I like to use struct variants

```rust
CreatedSchema {
    database_id: i64,
    schema_id: i64,
    schema_name: String,
}
```

whenever there are three elements in a tuple, unless the meaning of the components is _really_ idiomatic (e.g., a 3D point, or a differential (data, time, diff) entry). Even with 2-tuples it can be helpful sometimes!